netrd/distance/dmeasure.py

```diff
@@ -0,0 +1,282 @@
+"""
+d_measure.py
+--------------------------
+
+Distance measure based on the Jensen-Shannon Divergence
+between the network node dispersion distributions of two graphs.
+
+Schieber, T. A. et al.
+Quantification of network structural dissimilarities.
+Nat. Commun. 8, 13928 (2017).
+
+https://www.nature.com/articles/ncomms13928
+
+author: Brennan Klein
+email: brennanjamesklein@gmail.com
+Submitted as part of the 2019 NetSI Collabathon.
+
+"""
+
+import numpy as np
+import networkx as nx
+from collections import Counter
+from .base import BaseDistance
+
+# from ..utilities import entropy
```

Comment:
Please delete. Also why not use the one on utilities?

Reply:
Meant to open an issue about this - our entropy is for a set of values---it makes the histogram and then uses scipy entropy---but here we don't really have that. We should consider renaming our entropy to something like `entropy_from_values` for clarity, I think.

Reply:
Maybe just leave a comment saying just that here.

Reply:
It's a more intrusive change but my preference would be to rename `netrd.utilities.entropy.entropy`. After not using it for a while I was really confused when I tried to drop netrd `entropy` into this function.